src/app/public/modules/phone-field/phone-field.component.ts

```diff
@@ -104,6 +102,13 @@ import {
 })
 export class SkyPhoneFieldComponent implements OnDestroy, OnInit {
 
+  /**
+   * Specifies the
+   * [International Organization for Standardization Alpha 2](https://www.nationsonline.org/oneworld/country_code_list.htm)
```

Comment:
The link isn't displaying properly, and it's displaying the markdown.

Reply:
Where are you seeing that? When I follow the preview link, I see link text that works:
![image](https://user-images.githubusercontent.com/7850105/86803534-13293c00-c044-11ea-9e42-ace7ab4a17a8.png)

Reply:
Oops, I commented on the wrong line. It's actually under the interfaces section
![interfaces](https://user-images.githubusercontent.com/8514821/86804261-64353200-c03c-11ea-9cd0-2c129de93c65.png)

Reply:
I'm going to resolve this because my comment's misplaced. Added a comment in the correct section.